packages/neuron-ui/src/components/SUDTSend/hooks.ts

```diff
@@ -0,0 +1,198 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { AccountType, isAnyoneCanPayAddress, isSecp256k1Address, isSuccessResponse, shannonToCKBFormatter } from 'utils'
+import { SUDTAccount } from 'components/SUDTAccountList'
+import { DEFAULT_SUDT_FIELDS } from 'utils/const'
+import { generateChequeTransaction, generateSUDTTransaction, getSUDTCellExtraCkb } from 'services/remote'
+import { AppActions, useDispatch } from 'states'
+
+export enum SendType {
+  secp256Cheque = 'cheque',
+  secp256NewCell = 'secp256NewCell',
+  acpExistCell = 'acpExistCell',
+  acpNewCell = 'acpNewCell',
+  unknowNewCell = 'unknowNewCell',
+  sendCKB = 'sendCKB',
+}
+
+export enum AddressLockType {
+  secp256 = 'secp256',
+  acp = 'acp',
+  unknow = 'unknow',
+}
+
+export function useAddressLockType(address: string, isMainnet: boolean) {
+  return useMemo(() => {
+    if (isSecp256k1Address(address)) {
+      return AddressLockType.secp256
+    }
+    if (isAnyoneCanPayAddress(address, isMainnet)) {
```

Comment:
`PW Lock ACP Address` is missing here.

Ref: https://github.com/Magickbase/ckb-wallet-and-explorer/wiki/Neuron-Functionalities
Ref: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/.env#L30